Dockerfile

```diff
@@ -19,9 +19,23 @@
 # submit itself to any jurisdiction.
 
 FROM python:3.5
+
+RUN apt-get update && \
+    apt-get install -y vim emacs-nox
```

Comment:
do we consider by default that having some usable editor inside the containers is useful even for PROD?